helper/convert/value.go

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package convert
+
+import (
+	"github.com/hashicorp/go-cty/cty"
+	"github.com/hashicorp/terraform-plugin-go/tftypes"
+)
+
+func PrimitiveTfValue(in cty.Value) tftypes.Value {
+	if in.IsNull() {
+		return emptyTfValue(ToTfType(in.Type()))
+	}
+
+	var val tftypes.Value
+	switch in.Type() {
+	case cty.String:
+		val = tftypes.NewValue(tftypes.String, in.AsString())
+	case cty.Bool:
+		val = tftypes.NewValue(tftypes.Bool, in.True())
```

Comment:
This looks like it'll always return true, rather than whatever the boolean value is

Reply:
I originally thought so too, but 

```
// True returns true if the receiver is True, false if False, and panics if
// the receiver is not of type Bool.
```

Reply:
Lolol, what an odd name, I'm sure there's a fun history lesson on why that is named that way 😆